### PR TITLE
feat PUD-1231 (pci.storage.databases): add grafana engine

### PR DESCRIPTION
**Pull request by jperchoc**
PUD-1231

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/aiven-ga2` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #PUD-1231
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Adding Grafana engine to cloud databases

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html [hunk 1]
```diff
@@ -302,6 +302,26 @@
...
                 </oui-tile-definition>
+                <oui-tile-definition
+                    data-term="{{:: 'pci_databases_general_information_user_label' | translate }}"
+                    data-ng-if="$ctrl.isFeatureActivated('resetAdminUserFromDashboard')"
+                >
+                    <oui-tile-description>
+                        <span data-ng-bind="$ctrl.users[0].username"></span>
+                        <button
+                            class="oui-tile__button oui-button oui-button_ghost oui-button_block"
+                            data-ng-click="$ctrl.resetPassword()"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html:313]:
```suggestion
                        <button
                            type="button"                        
                            class="oui-tile__button oui-button oui-button_ghost oui-button_block"
                            data-ng-click="$ctrl.resetPassword()"
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -68,5 +68,8 @@
   "pci_databases_general_information_kafka_rest_api_enable": "Activer",
   "pci_databases_general_information_kafka_rest_api_disable": "Désactiver",
   "pci_databases_general_information_kafka_rest_service_uri_label": "URI du service REST Kafka",
-  "pci_databases_general_information_kafka_rest_api_enable_disable_error": "Une erreur est survenue lors de l'activation/désactivation de Kafka REST API."
+  "pci_databases_general_information_kafka_rest_api_enable_disable_error": "Une erreur est survenue lors de l'activation/désactivation de Kafka REST API.",
+  "pci_databases_general_information_reset_password": "Régénérer le mot de passe",
+  "pci_databases_general_information_reset_password_success": "Le mot de passe de l'utilisateur {{ user }} a été régénéré avec succès: <code>{{ password }}</code>. Veuillez le noter car il ne pourra pas être récupéré par la suite.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json:73]:
```suggestion
  "pci_databases_general_information_reset_password_success": "Le mot de passe de l'utilisateur {{ user }} a été régénéré avec succès : <code>{{ password }}</code>. Veuillez le noter car il ne pourra pas être récupéré par la suite.",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html]: Added: https://github.com/ovh/manager/pull/6376/commits/c216fcefcaa71dc9e0b306a04c2667fab5636544

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json]: Changed: https://github.com/ovh/manager/pull/6376/commits/c216fcefcaa71dc9e0b306a04c2667fab5636544

**Review by Jisay — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**